### DISCUSSION TITLE
problem in install!

### DISCUSSION THREAD

**Discussion by mrshab72** [ctu-mrs/mrs_uav_system#69] [General]
Hi MRS team
when I start automatic installation, I get this error:
```

E: Failed to fetch https://packagecloud.io/github/git-lfs/ubuntu/dists/focal/InRelease  403  Forbidden [IP: 143.204.170.173 443]
E: The repository 'https://packagecloud.io/github/git-lfs/ubuntu focal InRelease' is not signed.

```
what is the solution??
tanks for you

**Comment by spurnvoj** [comment]
Hi, it seems the server for `git-lfs` refused to connect you to their server. Try to call the following command in the terminal 
```bash
curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
```
If everything is alright, it should tell you `The repository is setup! You can now install packages`. After that try the installation of our system again.

**Reply by mrshab72** [reply]
thanks a lot, it work :+1:
